docs/trace/advanced/stratified-sampling-example/README.md

```diff
@@ -0,0 +1,98 @@
+# Stratified Sampling: An Example
+
+This is a proof of concept for how we can achieve stratified sampling in
+OpenTelemetry.NET.
+
+## What is stratified sampling?
+
+Stratified sampling is a way to divide a population into mutually exclusive
+sub-populations or "strata". For example, the strata for a population of
+"queries" could be "user-initiated queries" and "programmatic queries". Each
+stratum is then sampled using a probabilistic sampling method. This ensures
+that all sub-populations are represented.
+
+## How does this example do stratified sampling?
+
+We use disproportionate stratified sampling here - i.e., the sample size of
```

Comment:
lets describe the approach here in the doc - we achieve this by using a custom Sampler, which internally holds 2 samplers, which are invoked acording to condition.. 
Also list the pre-requisite for this to work - that the tag being used to make  decision should be provided as part of activty creation itself.

Reply:
Done, thanks!